src-electron/zcl/zcl-loader-silabs.js

```diff
@@ -304,10 +304,24 @@ function prepareAtomic(a) {
     size: a.$.size,
     description: a.$.description,
     isDiscrete: a.$.discrete == 'true',
+    isComposite: a.$.composite == 'true',
     isSigned: a.$.signed == 'true',
-    isString: a.$.string == 'true',
-    isLong: a.$.long == 'true',
-    isChar: a.$.char == 'true'
+    isString: a.$.string == 'true' || a.$.name.toLowerCase().includes('string'),
+    isLong:
+      a.$.long == 'true' ||
+      (a.$.name.toLowerCase().includes('long') &&
+        a.$.name.toLowerCase().includes('string')),
```

Comment:
This seems like it would trigger on names like "stringorlong"....

Reply:
Fixed